api/app/src/shared/analytics.ts

```diff
@@ -0,0 +1,32 @@
+import { PostHog } from "posthog-node";
+import { Request } from "express";
+
+import { Config } from "./config";
+import { getCxId } from "../routes/util";
+
+export const queryAnalytics = ({
+  message,
+  req,
+  apiType,
+}: {
+  message: string;
+  req: Request;
```

Comment:
Since we're only using `Request` to get the `cxId`, maybe we request the `cxId` on the function instead of Request? It would make it easier to use this function on contexts where we don't have the request.